src/switch_core_media.c

```diff
@@ -13126,6 +13126,12 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_receive_message(switch_core_se
 													   "codec_string", "=%s", switch_channel_get_variable(session->channel, "ep_codec_string"));
 				}
 				
+				/* Required fix to allow audio to be added to video
```

Comment:
`Required fix` sounds a bit off. If merged there is no issue to fix anymore.

Reply:
so, what exactly do you want me to change?

Reply:
Updated misleading comment